### ISSUE TITLE
Issue on docs: formatting isn't correct

### ISSUE THREAD

**Issue by nabinbhatt** [novuhq/docs#255]
Path: /quickstarts/angular

for example as you can see on the image, there shouldn't be line break, and also the word is not visible correctly at the end.
![image](https://github.com/novuhq/docs/assets/83168881/6f87e2cc-62b5-4c8b-8fd0-9dcecf1a9a2d)

**Comment by nabinbhatt** [comment]
@sumitsaurabh927 could you please take a look and assign this issue to me?

**Comment by unicodeveloper** [comment]
@nabinbhatt Can you find other issues and add to this issue?

**Comment by nabinbhatt** [comment]
Yes, the other pages encounter a similar issue, but it's mostly a matter of readability and doesn't have a significant impact.

for example in the path /quickstarts/vanillajs
![image](https://github.com/novuhq/docs/assets/83168881/97d07c9f-f04a-4430-b0bc-576daaa8406d)

**Comment by unicodeveloper** [comment]
I am asking because I won't accept any new issue concerning this type of problem. They are low hanging fruits that works even if nothing is changed on them @nabinbhatt

**Comment by unicodeveloper** [comment]
I have assigned you to the issue @nabinbhatt

**Comment by nabinbhatt** [comment]
Can you reopen #258 or should I raise a new PR for this? @unicodeveloper
